### ISSUE TITLE
Drummer stability re overwrites

### ISSUE THREAD

**Issue by scripting** [scripting/drummerRFC#22]
A [post on Scripting News](http://scripting.com/2022/02/24/170155.html?title=drummerStatusReport) about Drummer stability. 

Just wanted to provide a place for people to comment.

**Comment by akaKenSmith** [comment]
As noted in the update, I haven’t noticed any problems using the web version for a good long time…

**Comment by scotthansonde** [comment]
I also have not had any overwrites since scripting/drummerSupport#150 was fixed

**Comment by rtanglao** [comment]
OT but thanks for the software over the years dave!
BTW i have never seen overwrites but maybe i don't use Drummer enough?!?

**Comment by dmatchett** [comment]
<img width="1059" alt="Friday 3;59 PM" src="https://user-images.githubusercontent.com/52044648/155819001-8e258477-1e8e-4b6f-ac2a-07d18d3340ab.png">

**Comment by scripting** [comment]
@dmatchett — we’ll get this fixed. The first thing to do is to do a backup and put it in a safe place. In the morning I’ll find a way for you to get a clean slate with the tabs.

**Comment by scripting** [comment]
@dmatchett, et al --

I'm glad to get a chance to try to sweep aside all the broken tabs that may be left over from various bugs in Drummer in the past (knock wood). Here's what you do. If this works I'll put out a change note.

### How to test

1. Reload the Drummer app. Verify that the version is 2.0.18.

2. Choose _Download my files_ from the Tools menu to get a current copy of all your files on the server.

3. On a piece of paper write down the names of the files you want to re-open after starting over with open tabs. 

4. Choose _Reset tabs_ from the Tools menu. Confirm that you want to close all the tabs and reload the Drummer app. 

5. The screen clears, the program waits three seconds to be sure the new prefs have been saved to the server, then it reloads the Drummer app. You should see the Notes outline in the first and only open tab. You can now re-open the files you wrote down in an earlier step.

### Who should do this

Basically anyone who's here. I'd like to be sure it works on all your systems.

Let me know how it goes! :smile:

**Comment by scotthansonde** [comment]
I tried this out. I don't have any broken tabs, but with a number of open tabs I chose the Reset tabs command and all the tabs closed as expect. 

I'll certainly be using this feature regularly since I like to start an editing session with a clean slate.

**Comment by scripting** [comment]
@scotthansonde -- thanks for testing this out. Glad it worked.

I wonder what became of @dmatchett -- I need to know if this solved the problem on his setup before I can close this little project.

**Comment by dmatchett** [comment]
Yes, worked a treat. Sorry for the delay.

**Comment by scripting** [comment]
@dmatchett — thanks and glad to hear it. If there are other problems, please speak up. 😀

**Comment by PostMonsterG** [comment]
I just found the new reset tabs command, and it worked perfectly for me. Thank you very, very much.
